.github/workflows/linters.yml

```diff
@@ -6,11 +6,11 @@ on:
 
 jobs:
   PyLint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        os: [ubuntu-20.04]
-        python-version: [3.7, 3.8]
+        os: [ubuntu-20.04, ubuntu-22.04]
+        python-version: [3.8, 3.10.7]
```

Comment:
i opt: `python-version: ['3.8', '3.10']`

Reply:
Thanks, good catch! I got `unknown version` for 3.10 earlier, but I forgot that I hadn't stringified the version. Fixed now, and running on 3.10.